docs/intro.mdx

```diff
@@ -7,16 +7,9 @@ sidebar_position: 0
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-|Lab goals|Learn about Aggregation Pipelines writing Aggregation Pipelines|
+|Lab goals|Learn about aggregation pipelines|
 |-|-|
-|What you'll learn|What are Aggregation Pipelines |
-||How are they different from simple MongoDB queries or SQL|
-||How to use simple pipelines to get data, filter it, show the fields needed|
-||How to write more advanced pipelines, querying arrays, counting and sorting results|
-||How to do JOINs using $lookup|
-||How to group data|
-||Exporting data|
-||How to write aggregations using popular programming languages (TBD)|
+|What you'll learn| <ul><li>What an aggregation pipeline is</li><li>How aggregation pipelines are different from other MongoDB queries or SQL</li><li>How to use simple aggregation pipelines to get data, filter it, and show particular fields</li><li>How to write more advanced pipelines that query arrays, count documents, and sort the results</li><li>How to join collections using `$lookup`</li><li>How to group data</li><li>How to export data</li></ul>
```

Comment:
I don't have Docusaurus set up locally, so I'm not sure if this formatting will actually work. It looks fine in the GitHub preview.